packages/syft/src/syft/protocol/protocol_version.json

```diff
@@ -1,5 +1,1172 @@
 {
-  "1": {
-    "release_name": "0.9.1.json"
```

Comment:
@eelcovdw @abyesilyurt Its seems like we have reset the protocol file. I think we want to ensure compatibility with previous version ?